packages/core/styles/components/_toast.scss

```
@@ -111,55 +111,56 @@ ix-toast {
-  > aside[data-intent='neutral'] {
+  >aside[data-intent='neutral'] {
     border-left: 3px solid var(--ix-border);
   }
 
-  > aside[data-intent='gradient'] {
-    border-left: 3px solid var(--ix-gradient);
+  >aside[data-intent='gradient'] {
+    border: 1px solid #ffffff;
+    border-left: 3px solid #ffffff;
 
     background: var(--ix-gradient-bg);
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)